.gitignore

```diff
@@ -50,3 +50,4 @@ src/os/windows/ottdres.rc
 !/config.lib
 !*.in
 *.tmp
+*.swp
```

Comment:
Remember, that as `vim` user you can also add these to your global git ignore :)

Reply:
Those are vim swap files, they are not relevant to the actual code and I wanted them not to show up in the git status. I can remove this from the .gitignore as well if it's an issue.

Reply:
you can set a global gitignore file like this:
`git config --global core.excludesfile '~/.gitignore'`
there you can put all editor tempfiles

it doesn't make sense to put them into the repo, because everyone uses a different editor

Reply:
Thanks for the tip, I actually never thought of doing that.
I will revert this change then.